02-household-queries/chainlit-household-bot.py

```diff
@@ -2,15 +2,33 @@
 
 from datetime import date
 import pprint
+import shutil
 
 import chainlit as cl
 from chainlit.input_widget import Select, Switch, Slider
 
-from llm import ollama_client
+import chromadb
+from chromadb.config import Settings
+
+from langchain.chains import ConversationalRetrievalChain
+from langchain_community.embeddings import SentenceTransformerEmbeddings, HuggingFaceEmbeddings
+from langchain_community.vectorstores import Chroma
+from langchain.memory import ChatMessageHistory, ConversationBufferMemory
+from langchain_google_genai import GoogleGenerativeAIEmbeddings
+
+import os
+
+from ingest import add_json_html_data_to_vector_db, add_pdf_to_vector_db, ingest_call
+from llm import google_gemini_client, gpt4all_client, ollama_client
+from retrieval import retrieval_call
 
 OLLAMA_LLMS = ["openhermes", "llama2", "mistral"]
-OTHER_LLMS = ["someOtherLLM"]
+GOOGLE_LLMS = ["gemini-pro"]
+# GPT4ALL_LLMS = ["gpt4all"]
 
+GOOGLE_EMBEDDINGS=["models/embedding-001"]
```

Comment:
Rename to make it obviously from Google. Something like:
```suggestion
GOOGLE_EMBEDDINGS=["google::models/embedding-001"]
```
Then you can split on `::` to get the embedding name. Or use a space as the delimiter instead.